docs/_docs/modifiers/modifiers.md

```diff
@@ -122,6 +122,57 @@ called `artwork`, which is a nullable `android.graphics.Bitmap` instance. To use
 the background fill from the document, pass a value to replace the background
 fill or pass `null`.
 
+### Brush content {#brush-content}
```

Comment:
Wow documentation too!

Reply:
I'm living in @timothyfroehlich shadow ;) -- I just followed the amazing docs on testing to generate a baseline image for the shader test. Literally the only manual step I had to do was pull the new `dcf` file off the device, and that's already automated in the `feature/rustlayout` branch :).